msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -54,135 +62,195 @@ You must configure the `fetchTask` property under `bots.configuration` object in
+               }
+               return response;
+      ```
 
-   :::image type="content" source="../../assets/images/bots/set-up-a-bot.png" alt-text="Screenshot shows set up a bot in chat.":::
+   ---
 
-   The bot is installed in the chat.
+When a user reconfigures the bot, the `fetchTask` property in the app manifest file initiates `ConfigFetch` in the bot logic. The user can reconfigure the bot settings post-installation in two ways:
 
-### Reconfigure the bot
+* @mention the bot in the message compose area. Select the **Settings** option that appears above the message compose area. A dialog appears, update, or changes the bot's configuration settings in the dialog.
 
-To reconfigure the bot, follow these steps:
+   :::image type="content" source="../../assets/images/bots/reconfiguration-mention-bot.gif" alt-text="Screenshot shows the configuration option for the bot in the message compose area.":::
 
-1. Go to the chat and **@mention** the bot in the message compose area and select **Send**.
+* Hover over the bot, the bot profile card appears. To update or change the bot's configuration settings, select the settings icon in the bot profile card.
 
-   :::image type="content" source="../../assets/images/bots/mention-bot.png" alt-text="Screenshot shows the interaction of bot.":::
+   :::image type="content" source="../../assets/images/bots/reconfiguration-hover.gif" alt-text="Screenshot shows the configuration option for the bot in a Teams group chat.":::
 
-1. When you hover over the bot from the conversation, an Adaptive Card appears. Select the **Settings** icon in the Adaptive Card.
 
-   :::image type="content" source="../../assets/images/bots/bot-adaptive-card-interaction.png" alt-text="Screenshot shows the Adaptive Card with settings icon.":::
+## Best practices
 
-   A bot configuration Adaptive Card appears.
+* If you want to have an individual channel-level configuration of your bot, ensure that you track the configuration as per the channel. Configuration data isn't stored and the invoke payload includes the sufficient [channelData](../../messaging-extensions/how-to/action-commands/create-task-module.md#payload-activity-properties-when-a-dialog-is-invoked-from-a-group-chat).
 
-1. Reconfigure the bot settings and select **Submit**.
+* Provide a clear and user-friendly dialog that prompts the user to enter the required information for the bot to operate properly, such as a URL, an area path, or a dashboard link.
 
-   :::image type="content" source="../../assets/images/bots/reconfigure-bot-settings.png" alt-text="Screenshot shows the Adaptive Card with settings icon to reconfigure.":::
+* Avoid sending multiple notifications or requests for configuration after the installation, as it might confuse the users.
 
-   The bot sends a response message.
+* Single sign-on (SSO) is not supported.
 
 ## Code sample
 
-| **Sample name** | **Description** |**Node.js** |
+| **Sample name** | **Description** |**.NET** |**Node.js** |
```

Comment:
Let's add a manifest example link here too, like the other scenarios

Reply:
Updated.